save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/docker/DockerContainerManager.kt

```diff
@@ -66,6 +66,7 @@ class DockerContainerManager(
         val buildImageResultCallback: BuildImageResultCallback = try {
             val buildCmd = dockerClient.buildImageCmd(dockerFile)
                 .withBaseDirectory(tmpDir)
+                .withPlatform("linux/amd64")
```

Comment:
Does it essentially act as no-op when platform is the same as host platform?

Reply:
Well, `amd64` is, let's say, the most popular architecture. I mean mostly all the laptops and PCs have it. It seems like it should work the same as if `withPlatform` would not be present because `amd64` seems to be is a "default" architecture. And of cause we need linux.

Reply:
>  `amd64` seems to be is a "default" architecture

There can be a bunch of different images under the same tag but for different architectures, and what Docker does is it reads manifest file associated with the tag to find the image for the current host architecture. So on `linux/amd64` hosts setting `--platform linux/amd64` should indeed match the default behavior of Docker.